extensions/box2d.js

```diff
@@ -13147,6 +13340,23 @@
       body.ApplyTorque(-Cast.toNumber(args.force));
     }
 
+    setAngVelocity(args, util) {
+      let body = bodies[util.target.id];
+      if (!body) {
+        body = this.setPhysicsFor(util.target);
+      }
+      body.SetAngularVelocity(args.force);
```

Comment:
Using the pre-existing spin with force block, a positive value causes a clockwise rotation, but here a positive value means counterclockwise. The new blocks should match the original.

Reply:
How should I go about fixing this? Do I just invert the value of the argument input?